stable/ghost/templates/NOTES.txt

```diff
@@ -31,9 +32,9 @@ host. To configure Ghost with the URL of your service:
 {{- if eq .Values.serviceType "ClusterIP" }}
 
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
-  echo Blog URL  :  http://127.0.0.1:2368/
-  echo Admin URL : http://127.0.0.1:2368/ghost
-  kubectl port-forward $POD_NAME 2368:2368
+  echo Blog URL  :  http://127.0.0.1{{ if .Values.ghostPort }}:{{ .Values.ghostPort  }}{{ end }}/
```

Comment:
That service IP likely isn't accessible from outside the cluster though? I'm open to changing the way we describe how ClusterIP services can be accessed, but I would argue that this isn't a big issue given that the default service type is LoadBalancer (so you'd explicitly be choosing to use ClusterIP for some reason) and Ghost only runs in a single pod anyway.